src/coreclr/pal/src/configure.cmake

```diff
@@ -220,18 +220,6 @@ int main(void)
 check_cxx_source_runs("
```

Comment:
`SSCANF_SUPPORT_ll` can be deleted too.

Reply:
I see a possible reference to it here:
https://github.com/dotnet/diagnostics/blob/cc5eca29afa2d9cacf6700ddc8d843082b5955c6/src/shared/pal/src/cruntime/printf.cpp#L36-L40

Not sure whether line 37 or 39 should be preserved. Can you advise?

Reply:
You link is for the PAL clone in dotnet/diagnostic repo. It may be still needed there.

The printf.cpp file does not exist in dotnet/runtime repo. We have been actively working on eliminating the PAL in dotnet/runtime repo. Many files that still exist in other PAL clones have been deleted in dotnet/runtime.

Reply:
> You link is for the PAL clone in dotnet/diagnostic repo. It may be still needed there. ... Many files that still exist in other PAL clones have been deleted in dotnet/runtime.

Right. I was concerned that if I deleted that clause from the runtime repo's cmake file, it would automatically be synced to the diagnostics repo's cmake file, potentially causing errors in that repo.

Are you saying that I don't need to worry about this? It's safe for me to delete the clause from the runtime repo's cmake file, and it won't negatively impact the diagnostics repo?

Reply:
It is safe to delete the clause from the runtime repo.

The PAL sync into diagnostic repo is done manually once in a blue moon. The next one will have to deal with printf.cpp deletion where `SSCANF_SUPPORT_ll` was used.